system/classes/route.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * route class which separates the controller, methods and parameters from
+ * the url
+ * 
+ */
+
+class route 
```

Comment:
As per standards the curly braces `{` shouldn't start from the next line.It should be `class route {`